install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -17,6 +17,6 @@ data:
       "baremetalIronic": "quay.io/openshift/origin-ironic:v4.2.0",
       "baremetalIronicInspector": "quay.io/openshift/origin-ironic-inspector:v4.2.0",
       "baremetalIpaDownloader": "quay.io/openshift/origin-ironic-ipa-downloader:v4.2.0",
-      "baremetalRhcosDownloader": "quay.io/openshift/origin-ironic-rhcos-downloader:v4.2.0",
+      "baremetalMachineOsDownloader": "quay.io/openshift/origin-ironic-machine-os-downloader:v4.3.0",
```

Comment:
Should the version for ironic-machine-os-downloader be 4.2 ? The fix in #454 used 4.3 since the understanding at that time was it would be merged to master and 4.3-release. If it is being backported to 4.2, then the image tag here shouls probably be v4.2.0.